snapcraft/cli/assertions.py

```diff
@@ -0,0 +1,112 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import sys
+
+import click
+
+import snapcraft
+from snapcraft.storeapi import errors
+from . import echo
+
+
+@click.group()
+def assertionscli():
+    """Store assertion relevant commands"""
+    pass
+
+
+@assertionscli.command('list-keys')
+def list_keys():
+    """List the keys available to sign assertions.
+
+    This command has an alias of `keys`.
+    """
+    try:
+        snapcraft.list_keys()
+    except Exception as e:
+        echo.error(e)
+        sys.exit(1)
```

Comment:
Given that each is an independent command that can be executed on its own, I'd say no... it really isn't harder to test. This also allows for fine grained exception catching to improve the product